src/System.IO.FileSystem/tests/FileInfo/Open.cs

```diff
@@ -12,6 +12,40 @@ protected override FileStream CreateFileStream(string path, FileMode mode)
         {
             return new FileInfo(path).Open(mode);
         }
+
+        [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework, "FileInfo.Open(string, filemode) on netfx always uses FileAccess.ReadWrite instead of choosing a FileAccess based on the FileMode. This bug was fixed in netcoreapp.")]
```

Comment:
I don't personally think we should. There are too many inconsistencies requiring too many bugs to open across the framework. And netfx-port-consider is generally applied to closed issues which we do not have for these  bugs, so I guess we would open new issues just to close them and mark them? Seems like too much. I think just having this tag and the associated comment is adequate for now, and if/when we make an IO backport push we can easily find the bugs that have been fixed in netcoreapp but not netfx by searching for these attributes.